### DISCUSSION TITLE
Location of change theme button

### DISCUSSION THREAD

**Discussion by Zigolox** [digitalungdom-se/digitalungdom.se#114] [General]
Right now the placement of the change theme button might be too central. Other websites puts the button more hidden. Se following examples: 

Github:
![image](https://user-images.githubusercontent.com/41542666/103215027-766c9a80-4912-11eb-8cd6-fef689fb9873.png)

Youtube:
![image](https://user-images.githubusercontent.com/41542666/103215073-98feb380-4912-11eb-8d7a-e884e0d88c84.png)

Reddit:
![image](https://user-images.githubusercontent.com/41542666/103215102-b59aeb80-4912-11eb-90f0-3244ac8e7ac3.png)

**Comment by Zigolox** [comment]
Right now our change theme is besides the notification button:

![image](https://user-images.githubusercontent.com/41542666/103215249-1b877300-4913-11eb-9f66-94f3faba6ea4.png)

**Comment by charlesmaddock** [comment]
I think the current placement is fine for the first version
